charts/internal/seed-controlplane/charts/csi-driver-controller/templates/configmap-monitoring.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: csi-driver-controller-monitoring-config
```

Comment:
Is this really monitoring config?

Reply:
Maybe also could be renamed to "observability"?